client/README.md

```diff
@@ -28,3 +28,10 @@ export default {
 - Replace `plugin:@typescript-eslint/recommended` to `plugin:@typescript-eslint/recommended-type-checked` or `plugin:@typescript-eslint/strict-type-checked`
 - Optionally add `plugin:@typescript-eslint/stylistic-type-checked`
 - Install [eslint-plugin-react](https://github.com/jsx-eslint/eslint-plugin-react) and add `plugin:react/recommended` & `plugin:react/jsx-runtime` to the `extends` list
+
+## Running with Docker Compose
+
+1. Make sure you have Docker installed on your machine.
+2. Navigate to the root directory of the project.
+3. Run `docker-compose up` to start the frontend services.
+4. Access the client application at `http://localhost:4200`.
```

Comment:
It doesn't look like we're using the correct setup for `docker-compose` with our two directories, `client/` and `server/`. Please use the following references to do this:
1. [Github Project File Example](https://github.com/adrianhajdin/project_ai_mern_image_generation/pull/51/files)
2. [Let us take an example application with the following structure:](https://www.docker.com/blog/docker-compose-from-local-to-amazon-ecs/)
3. [Using docker-compose](https://www.baeldung.com/ops/multiple-dockerfiles#docker-compose)

Reply:
We never updated the docs with our changes here @haydenbalsys